include/GLFW/glfw3.h

```diff
@@ -1272,6 +1272,57 @@ extern "C" {
 #define GLFW_HAND_CURSOR            GLFW_POINTING_HAND_CURSOR
 /*! @} */
 
+/*! @addtogroup window
+ *  @{ */
+/*! @brief Disable the progress bar.
+ *
+ *  Disable the progress bar.
+ *
+ *  Used by @ref window_taskbar_progress.
+ */
+#define GLFW_TASKBAR_PROGRESS_NOPROGRESS    0
```

Comment:
I think this is better described as `GLFW_TASKBAR_PROGRESS_DISABLED`.

Reply:
Done 3e7b016.